netty/src/test/java/io/grpc/netty/NettyClientTransportTest.java

```diff
@@ -538,6 +548,105 @@ public void maxHeaderListSizeShouldBeEnforcedOnClient() throws Exception {
     }
   }
 
```

Comment:
I'll take another pass at stabilizing these tests 🙂

Reply:
It's only a little off.
```
io.grpc.netty.NettyClientTransportTest > huffmanCodingShouldNotBePerformed FAILED
    expected: 1596
    but was : 1627
```

A BDP PING (or ACK) would be 17 bytes. And a SETTINGS ACK for initial window size changing would be 14 byte. Passing `autoFlowControl = false` to NettyClientTransport would avoid those... from the client. You can just make yet another `newTransport()` override. You can do the same for `newServer()`. (The other option is to add a mutable field to the test `autoFlowControl = true` that is read by `newTransport()`/`newServer()` and set it to false in your one test. Ideally we'd use the builder, but that's a larger conversion.

Reply:
`autoFlowControl` was already off on both sides. While trying to debug the differences, I disabled TLS and the tests immediately stabilized.

With the change to plaintext traffic, I rewrote the test to directly assert on the bytes written in the header - which is a better test *and* avoids the potential instability.